dataprofiler/tests/labelers/test_base_model.py

```diff
@@ -96,25 +96,98 @@ def test_set_parameters(self):
         with self.assertRaisesRegex(ValueError, 'test'):
             base_model.BaseModel.set_params(mock_model, **params)
 
-    def test_set_label_mapping_parameters(self):
+    def test_add_labels(self):
 
         # setup mock
         mock_model = mock.Mock(spec=base_model.BaseModel)
```

Comment:
This is a lot to mock. Can we mock the tensorflow model instead of the whole baseModel class?